### PR TITLE
rune: drop unnecessary dependency in README and dist/rpm

### PR DESCRIPTION
**Pull request by hustliyilin**
Signed-off-by: Yilin Li <YiLin.Li@linux.alibaba.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,22 +55,21 @@ Please refer to [Inclavare Containers Roadmap](ROADMAP.md) for the details. This
...
 
-2. Prepare the tools required by Inclavare Containers
-```shell
-cd inclavare-containers
-
-# install Go protobuf plugin for protobuf3
-go get github.com/golang/protobuf/protoc-gen-go@v1.3.5
-```
+2. Install the Dependency required by Inclavare Containers
+- Go 1.14.x or above.
```
Comment (jiazhang0) [inline README.md:66]: Change to:
```
Go version 1.14 or higher.
```

File: rune/dist/rpm/rune.spec [hunk 2]
```diff
@@ -61,6 +56,9 @@ install -p -m 644 %{name}/LICENSE %{buildroot}%{_defaultlicensedir}/%{name}-%{ve
 %{BIN_DIR}/%{name}
 
 %changelog
+* Sat Nov 21 2020 Yilin Li <YiLin.Li@linux.alibaba.com> - 0.5.0-1
+- Drop unnesseccary dependency.
```
Comment (jiazhang0) [inline rune/dist/rpm/rune.spec:60]: typo. Should be unnecessary.

**Review by jiazhang0 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -55,22 +55,21 @@ Please refer to [Inclavare Containers Roadmap](ROADMAP.md) for the details. This
...
 ```
 
-2. Prepare the tools required by Inclavare Containers
-```shell
-cd inclavare-containers
-
-# install Go protobuf plugin for protobuf3
-go get github.com/golang/protobuf/protoc-gen-go@v1.3.5
-```
+2. Install the Dependency required by Inclavare Containers
```
Comment (jiazhang0) [inline README.md:65]: rune/README.md also needs this modification.

Comment (hustliyilin) [reply README.md:65]: Done

**Pull request merged**